src/Umbraco.Web.UI.Client/src/views/common/dialogs/content/edit.html

```diff
@@ -12,15 +13,33 @@
 		</div>
 
 		<div class="umb-panel-body with-footer">
-            <div id="tab{{tab.id}}" ng-repeat="tab in content.tabs">
-                <div class="umb-pane" ng-if="!tab.hide">
-                    <h5>{{tab.label}}</h5>
-                    <umb-property property="property"
-                                  ng-repeat="property in tab.properties">
-                        <umb-editor model="property"></umb-editor>
-                    </umb-property>
-                </div>
-            </div>    
+
+            <div class="umb-pane" ng-if="content">
```

Comment:
Nice change here to the proper use of our own directives/components to make it a tabbed UI as opposed to the previous long list of items